apps/prairielearn/src/ee/pages/instructorAiGenerateJobReview/instructorAiGenerateJobReview.html.ts

```diff
@@ -0,0 +1,70 @@
+import { formatDate } from '@prairielearn/formatter';
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { HeadContents } from '../../../components/HeadContents.html.js';
+import { JobStatus } from '../../../components/JobStatus.html.js';
+import { Job } from '../../../lib/db-types.js';
+
+export function AiGenerateJobReviewPage({
+  resLocals,
+  genJobs,
+}: {
+  resLocals: Record<string, any>;
+  genJobs: Job[];
+}) {
+  return html` <!doctype html>
+    <html lang="en">
+      <head>
+        ${HeadContents({ resLocals })}
+      </head>
+      <body hx-ext="loading-states">
+        ${renderEjs(import.meta.url, "<%- include('../../../pages/partials/navbar'); %>", {
+          navPage: 'course_admin',
```

Comment:
Is this accurate given that you've also mounted this page on the course instance route?

Reply:
i'm not sure that `instance_admin` is the right menu for questions though? the questions general page also uses the `course_admin` menu afaik. changing to the `instance_admin` menu generates the following header:
![image](https://github.com/user-attachments/assets/dbb5fdb1-b662-4c4b-94a4-2ddea79d0a45). Should I keep that for instance route pages?